packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -276,6 +299,84 @@ const Updates = decorate([
           </Card>
         </Col>
+        <Col mediumSize={6}>
+          <Card>
+            <CardHeader>{_('releaseChannels')}</CardHeader>
+            <CardBlock>
+              <div>
+                {_('selectedChannel')}{' '}
+                <h5 className='d-inline-block'>
+                  <span className='tag tag-pill tag-info'>
+                    {xoaConfiguration.channel}
+                  </span>
+                </h5>
+              </div>
+              <br />
+              <form id='releaseChannelsForm' className='form'>
+                <div className='form-group'>
+                  <Select
+                    clearable={false}
+                    onChange={effects.onChannelChange}
+                    options={[
+                      ...map(xoaReleaseChannels, elt => ({
+                        label: elt.description,
+                        value: elt.id,
+                      })),
+                      {
+                        label: (
+                          <span className='font-italic'>custom channel</span>
+                        ),
+                        value: 'custom channel',
+                      },
+                    ]}
+                    placeholder={formatMessage(messages.selectChannel)}
+                    required
+                    value={helper(state, xoaConfiguration, 'channel')}
+                    disabled={state.addPrivateChannel}
+                  />
+                  <br />
+                  {state.addPrivateChannel && (
+                    <div>
+                      <div className='form-group'>
+                        <input
+                          autoFocus
+                          className='form-control'
+                          name='privateChannel'
+                          onChange={effects.linkState}
+                          placeholder={formatMessage(
+                            messages.privateChannelName
+                          )}
+                          required
+                          type='text'
+                          value={helper(
+                            state,
+                            xoaConfiguration,
+                            'privateChannel'
+                          )}
+                        />
+                      </div>
+                      <Button
+                        btnStyle='light'
+                        onClick={effects.cancelPrivateChannelAdd}
+                      >
+                        <Icon fixedWidth icon='remove-tag' />{' '}
+                        {_('genericCancel')}
+                      </Button>
+                    </div>
+                  )}
+                </div>{' '}
+                <ActionButton
+                  form='releaseChannelsForm'
+                  icon='success'
+                  btnStyle='primary'
+                  handler={effects.configure}
+                >
+                  {_('changeChannel')}
+                </ActionButton>
```

Comment:
Why is it necessary? Isn't enough for the user to select another channel?

Reply:
You're wright. but when the user select a custom channel, this action needs to be validated by a button

Reply:
Sorry, my question was for the `Cancel` button.